meta_model_building/utility.py

```diff
@@ -0,0 +1,71 @@
+""" This module contains methods called in the main script for obtaining data:
+get_opt_params.py"""
+
+import copy
+from tpot.config.regressor import regressor_config_dict
+from tpot.config.regressor_light import regressor_config_dict_light
+
+# model lists and dicts
+implemented_model_list = [
```

Comment:
You may want to make these tuples if you're not planning on modifying them at runtime (since they're essentially hard-coded here).